platform/ios/PolyPodApp/PolyPodTests/PolyOutTests.swift

```diff
@@ -146,4 +146,54 @@ class PolyOutTests: XCTestCase {
         
         wait(for: [expectation], timeout: 1.0)
     }
+    
+    func testImportOneArchive() {
+        let polyOut = PolyOut(session: NetworkSessionMock())
+        
+        let bundle = Bundle(for: type(of: self))
+        polyOut.activeFeature = Feature(path: URL(string: bundle.bundlePath)!, name: "Test", author: nil, description: nil, thumbnail: nil, thumbnailColor: nil, primaryColor: nil, links: nil)
+        
+        let url = bundle.url(forResource: "testZip", withExtension: "zip")!
+        
+        let expectation = XCTestExpectation(description: "Exp")
+        polyOut.importArchive(url: url.absoluteString, destUrl: nil) { newUrl in
+            XCTAssertTrue(newUrl != nil && newUrl != "", "newUrl is nil or empty")
+            polyOut.readDir(url: newUrl!) { stuff, error in
+                XCTAssertNil(error, "error is not nil")
+                XCTAssertTrue(stuff != nil && !stuff!.isEmpty, "no files were found")
+                XCTAssertTrue(stuff!.filter{ $0["path"] == "testZip/testfile.rtf" }.count > 0, "file not found")
+                expectation.fulfill()
+            }
+        }
+        wait(for: [expectation], timeout: 5.0)
+    }
+    
+    func testImportMultipleArchives() {
+        let polyOut = PolyOut(session: NetworkSessionMock())
+        
+        let bundle = Bundle(for: type(of: self))
+        polyOut.activeFeature = Feature(path: URL(string: bundle.bundlePath)!, name: "Test", author: nil, description: nil, thumbnail: nil, thumbnailColor: nil, primaryColor: nil, links: nil)
+        
+        let url1 = bundle.url(forResource: "multipleZips1", withExtension: "zip")!
+        let url2 = bundle.url(forResource: "multipleZips2", withExtension: "zip")!
+        
+        let expectation = XCTestExpectation(description: "Exp")
+        polyOut.importArchive(url: url1.absoluteString, destUrl: nil) { newUrl1 in
+            XCTAssertTrue(newUrl1 != nil && newUrl1 != "", "newUrl1 is nil or empty")
+            
+            polyOut.importArchive(url: url2.absoluteString, destUrl: newUrl1!) { newUrl2 in
+                XCTAssertTrue(newUrl2 != nil && newUrl2 != "", "newUrl2 is nil or empty")
+                XCTAssertTrue(newUrl1 == newUrl2)
+                
+                polyOut.readDir(url: newUrl1!) { stuff, error in
+                    XCTAssertNil(error, "error is not nil")
+                    XCTAssertTrue(stuff != nil && !stuff!.isEmpty, "no files were found")
+                    XCTAssertTrue(stuff!.filter{ $0["path"] == "multipleZips1/file1.rtf" }.count > 0, "file1  not found")
+                    XCTAssertTrue(stuff!.filter{ $0["path"] == "multipleZips2/file2.rtf" }.count > 0, "file2  not found")
+                    expectation.fulfill()
+                }
+            }
+        }
+        wait(for: [expectation], timeout: 20.0)
```

Comment:
Nit:
Tests will be easier to read if assertions are not done in completion blocks. I mean something like this:
```swift
let newURL1 = importArchive(url: url1.absoluteString, polyOut: polyOut)
XCTAssertTrue(newUrl1 != nil && newUrl1 != "", "newUrl1 is nil or empty")
let newURL1 = importArchive(url: url2.absoluteString, polyOut: polyOut)
XCTAssertTrue(newUrl2 != nil && newUrl2 != "", "newUrl2 is nil or empty")
```